### PR TITLE
change content of gitflowwithoutgitflow

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr Please review

File: index.html [hunk 1]
```diff
@@ -56,6 +56,7 @@
             <ul>
               <li><a href="/waysofworking/systemsetup.html">System setup</a></li>
               <li><a href="/waysofworking/git.html">Working with git</li>
+              <li><a href="/waysofworking/gitflowwithoutgitflow">Git flow without git-flow</li>
```
Comment (gsvr) [inline index.html]: needs trailing .html on filename

File: waysofworking/gitflowwithoutgitflow.html [hunk 2]
```diff
@@ -1,125 +1,50 @@
...
+    <p>(now keep an eye on the website for travis being happy)
+    (once travis is happy, merge in the feature branch)</p>
+    <pre><code>$ git checkout develop
+    $ git merge --no-ff feature/issue-xx-abcd-efgh
+    $ git branch -d feature/issue-xx-abcd-efgh
+    </code></pre>
+    <p>(then push your local develop to origin)</p>
+    <pre><code>$ git push</code></pre>
+
+</section>
```
Comment (gsvr) [inline waysofworking/gitflowwithoutgitflow.html]: no newline

**Comment by gsvr** [comment]
:+1:

**Pull request merged**